src/geo/gmaps/gmaps-default-map-style.js

```diff
@@ -1,69 +0,0 @@
-var DEFAULT_MAP_STYLE = [{
```

Comment:
Why have you removed this? There will not be an option to create a google map instead a leaflet one?